.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: c
+sudo: required
+dist: trusty
+matrix:
```

Comment:
This is a CI that will run against this repo, will compile zrpcd in Ubuntu and is for the benefit of the zrpcd development process, making it easy to find changes that break the build. Apex/OPNFV will keep doing its own functional tests, regardless of whether this is merged or not.

I say proposal in the sense that here is a working & mergeable proof of concept of a feature. If you want it, merge it, but if not please discuss with me what alternatives are there for you, or even if you care about this at all.